src/js/handler/month/core.js

```diff
@@ -8,6 +9,7 @@ var util = require('tui-code-snippet');
 var common = require('../../common/common'),
     domutil = require('../../common/domutil'),
     domevent = require('../../common/domevent');
+var datetime = require('../../common/datetime');
```

Comment:
To declare `datetime` from require() is not necessary `var` keyword. It can be upper line.